pages/docs/tooling/tooling-config-visualization.md

```diff
@@ -26,6 +26,13 @@ Please first install the dependencies:
 
 * `python3` and `pipx` (or `pip`)
 * [`graphviz`](https://graphviz.org/download/) for rendering the result.
+* [`cairo`](https://cairographics.org/) and [`gir](https://developer.gnome.org/documentation/guidelines/programming/introspection.html) for the GUI.
```

Comment:
```suggestion
* dependencies of [`pygobject`](https://gnome.pages.gitlab.gnome.org/pygobject/getting_started.html)
```

Reply:
The linked documentation is clear enough, so this can indeed be offloaded.